storage/sql/storage.go

```diff
@@ -326,6 +329,14 @@ func (conn *Connection) DeleteInstance(instance *models.Instance) error {
 	return tx.Commit().Error
 }
 
+func (conn *Connection) TruncateAll() {
+	tx := conn.db.Begin()
+	tx.Exec("delete from users")
```

Comment:
am I wrong, but can't the tables have prefixes?

Reply:
You are correct. I'll fix.

Reply:
that was only a hack to put several projects in the same database to pay less for heroku 😭 I'm ok with removing the prefixes support and stick to standard naming rules.

Reply:
Well, we have production data using a prefix of `production` :(

Reply:
it will make migrations easier too - they don't seem to love the prefix stuff either

Reply:
we could write a migration to move those tables?

Reply:
It would have to be an offline migration